src/material/button/_button-theme.scss

```diff
@@ -38,8 +33,44 @@
   ));
 }
 
+@function _get-text-button-color-tokens-for-palette($palette) {
+  @return (
+    focus-label-text-color: theming.get-color-from-palette($palette),
```

Comment:
Given that all of these are the same, maybe we should move `theming.get-color-from-palette($palette)` into a local variable?